src/components/new-request/search.vue

```diff
@@ -341,6 +359,16 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin, Sear
       this.isAlterOnline(this.getConversionType)
     ) return true
 
+    if (this.isChangeName &&
+      this.isNumberedCompany &&
+      !this.isSupportedChangeName(this.getEntityTypeCd)
+    ) return true
+
+    if (this.isChangeNameXpro &&
+      this.isXproFlow &&
+      this.isFederal
+    ) return true
```

Comment:
Is line 368 redundant (because line 367 already checks for xpro)?

What if the current jurisdiction is not federal?

Do you need to check the FF here also?

Reply:
^^ name search isn't working for me on temp URL

Also, can you show an example of a Federal name change?

Reply:
Federal:

![image](https://github.com/bcgov/namerequest/assets/116035339/2c9e7fba-4a19-4afa-a1bb-f0c41cc00433)

Reply:
Federal looks good 👍

Reply:
> ^^ name search isn't working for me on temp URL

That's because I was not logged in, and the not-logged-in-business-search isn't working right now (Vysakh is working on it).

So not an issue!